SonarQube.Client/Api/V5_00/GetSourceCodeRequest.cs

```diff
@@ -0,0 +1,39 @@
+﻿/*
+ * SonarQube Client
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using Newtonsoft.Json;
+using SonarQube.Client.Requests;
+
+namespace SonarQube.Client.Api.V5_00
```

Comment:
Fortunately, both SC and SQ have the same API currently.

SQ v5.0 is the earliest supported version. Even though the only consumer of this API for now will be by the taint feature which requires v8.x, I didn't see a reason to only make this API available in newer versions of SonarQube.